service/src/main/resources/application.yml

```diff
@@ -141,6 +141,32 @@ imputation:
   memoryRetryMultiplier: 2.0
 
 pipelines:
+  configurations:
+    pipelines:
+      array_imputation:
```

Comment:
nit, but can these be camelCase like the variables so that we don't have horrific mixed camel and snake case in the java (e.g. `getArray_imputation()`)

Reply:
yea im not sure why i did it this way, just typing stuff